src/service/mercury/index.ts

```diff
@@ -358,11 +380,35 @@ export class MercuryClient {
     }
   };
 
-  getAccountHistory = async (pubKey: string) => {
+  getAccountHistoryHorizon = async (pubKey: string, network: NetworkNames) => {
+    try {
+      const networkUrl = NETWORK_URLS[network];
+      const server = new Horizon.Server(networkUrl, {
+        allowHttp: !networkUrl.includes("https"),
+      });
+      const data = await fetchAccountHistory(pubKey, server);
+      return {
+        data,
+        error: null,
+      };
+    } catch (error) {
+      this.logger.error(error);
+      const _error = JSON.stringify(error);
+      this.logger.error(_error);
```

Comment:
I see that you have an established pattern of logging the raw error and then a string version of the error. I think that makes sense when the string version has some added info to make it more human readable. In this case, would this be a little noisy? Basically just seeing the same error twice in a row?

Reply:
ah I see yeah I think this is because the router has a built in logger that logs any uncaught errors in the route itself, so yeah maybe it makes sense to not log these in the services at all, good call.

Reply:
oh wait I see now that this one is just a copy pasta, removed in ce46724b35804ae35e3780f56c90bcc180227230